plugins/kubernetes/app/views/samson_kubernetes/_build_new.erb

```diff
@@ -1,6 +1 @@
-<div class="form-group">
-  <%= form.label :kubernetes_job, "Use Remote Kubernetes Job?", class: "col-lg-2 control-label" %>
-  <div class="col-lg-4">
-    <%= form.check_box :kubernetes_job %>
-  </div>
-</div>
+<%= form.input :kubernetes_job, as: :check_box, label: 'Use Remote Kubernetes Job?' %>
```

Comment:
yes, plugin views work by giving a partial to render ... just passing a block is not supported ... also keeps it consistent